crates/symbolicator-service/src/services/download/sentry.rs

```diff
@@ -98,37 +97,25 @@ impl SentryDownloader {
     ///
     /// If there are cached search results this skips the actual search.
     async fn cached_sentry_search(&self, query: SearchQuery) -> CacheEntry<Vec<SearchResult>> {
-        if let Some((created, entries)) = self.index_cache.lock().unwrap().get(&query) {
-            if created.elapsed() < self.cache_duration {
-                return Ok(entries.clone());
-            }
-        }
-
-        tracing::debug!(
-            "Fetching list of Sentry debug files from {}",
-            &query.index_url
-        );
+        let query_ = query.clone();
```

Comment:
@Swatinem Do you see a way to avoid this clone under a new name?

Reply:
not really, except creating a block just to create the future, though I wanted to explicitly avoid excessive rightward drift